src/least_cost_path/least_cost_path.py

```
@@ -285,129 +316,516 @@
+        water_raster_layer: QgsRasterLayer,
+        feedback = None
+    )-> tuple[QgsVectorLayer, QgsVectorLayer]:
+    # Инициализация прогресса
+    # ?progress = ProgressManager(
+    #     title="Анализ оптимальных путей", label="Инициализация..."
+    # ) 
+    # ?progress.init_progress(100)
+
+    # Переменные для хранения сообщений
+    height_message = None
+    rivers_message = None
```

Ruff (0.11.9)

Local variable `rivers_message` is assigned to but never used

Remove assignment to unused variable `rivers_message`

(F841)